packages/grafana-data/src/types/dataFrame.ts

```diff
@@ -244,6 +244,11 @@ export interface DataFrame extends QueryResultBase {
   length: number;
 }
 
+// Data frame that include aggregate value, for use by timeSeriesTableTransformer / chart cell type
+export interface DataFrameWithValue extends DataFrame {
+  value: number | null;
+}
+
```

Comment:
@ryantxu WDYT?

Reply:
I think this makes sense. I think the type should probably allow for string values and booleans, not inconceivable to want a sparkline of of booleans states :)

Reply:
Maybe idea for the future :) currently sparkline component only supports numbers

Reply:
Ah, let's wait then :)